docs/release_notes.rst

```diff
@@ -20,6 +20,10 @@ New Features
 
 * Enable full toml configuration and pyproject.toml (#534).
 
+Bug Fixes
```

Comment:
Can you move this up to Current dev version?

Reply:
Done